src/irmin-pack/unix/inode.ml

```diff
@@ -56,4 +56,16 @@ struct
   let v = Pack.v
   let integrity_check = Pack.integrity_check
   let purge_lru = Pack.purge_lru
+  let key_of_offset = Pack.key_of_offset
+
+  let unsafe_find_no_prefetch t key =
+    match Pack.unsafe_find_no_prefetch t key with
+    | None -> None
+    | Some v ->
+        let find ~expected_depth:_ k =
+          (* dead code *)
+          Pack.unsafe_find ~check_integrity:false t k
```

Comment:
Why is this dead code? Would it be better to `assert false` if so?

Reply:
Hm yes, it was an `assert false` during my testing!.. The placeholder code is the same as `unsafe_find` and should not cause any issue if it happens that I missed something, I still have to investigate more to be 100% sure it's really dead :)

Reply:
Gotcha. Two thoughts for how to address this:

1. If we can justify it is dead code in all scenarios, then we change this to `assert false` with an associated comment explaining why.
2. Otherwise, I'd say the simple tweak is to change this to a `TODO` comment to investigate in the future.